easybuild/easyconfigs/p/PyTorch-bundle/PyTorch-bundle-2.1.2-foss-2023a-CUDA-12.1.1.eb

```
@@ -0,0 +1,104 @@
+    }),
+    ('torchdata', '0.7.1', {
+        'preinstallopts': "USE_SYSTEM_LIBS=1 ",
+        'source_urls': ['https://github.com/pytorch/data/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['ef9bbdcee759b53c3c9d99e76eb0a66da33d36bfb7f859a25a9b5e737a51fa23'],
+        'runtest': False,  # circular test requirements
+    }),
+    ('torchtext', '0.16.2', {
+        'patches': ['torchtext-0.14.1_use-system-libs.patch'],
+        'source_urls': ['https://github.com/pytorch/text/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': [
+            {'torchtext-0.16.2.tar.gz': '6574b012804f65220329a2ad34a95c18e4df0d4cff2f862fb7862d57b374b013'},
+            {'torchtext-0.14.1_use-system-libs.patch':
+             '1366d10c4755b6003194f7313ca11d165a80a13d325bee9d669ea2b333d82536'},
+        ],
+        'runtest': 'pytest',
+        'testinstall': True,
+    }),
+    ('torchvision', '0.16.2', {
+        'preinstallopts': 'WITH_CUDA=1',
+        'installopts': '-v',
+        'patches': ['torchvision-0.16.2_ffmpeg-6.0-fix.patch'],
+        'source_urls': ['https://github.com/pytorch/vision/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': [
+            {'torchvision-0.16.2.tar.gz': '8c1f2951e98d8ada6e5a468f179af4be9f56d2ebc3ab057af873da61669806d7'},
+            {'torchvision-0.16.2_ffmpeg-6.0-fix.patch':
+             'a49336e7bfa1c950e886852bff37a3ea2146ac7bda87241e3ffb31c5cb869cce'},
+        ],
+        'runtest': 'pytest',
+        'testinstall': True,
+    }),
+    ('torchaudio', '2.2.1', {
+        'installopts': '--no-use-pep517 -v',
+        'patches': ['torchaudio-2.2.1_use-external-sox.patch'],
+        'preinstallopts': ('rm -r third_party/{sox,ffmpeg/multi} &&'
+                           ' USE_CUDA=1 USE_OPENMP=1 USE_FFMPEG=1 FFMPEG_ROOT="$EBROOTFFMPEG"'),
+        'source_urls': ['https://github.com/pytorch/audio/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': [
+            {'torchaudio-2.2.1.tar.gz': '5f2a9cda9369ff348d1d7ca2c0a1f01afc9ebff1a13df9775c22aa5c43f5b0f1'},
+            {'torchaudio-2.2.1_use-external-sox.patch':
+             'fc97b2bdf0ab16b9c4706bf2a31dd72d0ecb09e4af2f9c4a34e18fca8e0d165d'},
+        ],
+        'runtest': 'pytest',
+        'testinstall': True,
+    }),
+    ('pytorch-ignite', '0.4.13', {
+        'modulename': 'ignite',
+        'source_urls': ['https://github.com/pytorch/ignite/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['bfe4b6f1cd96e78c021a65a0c51350cdb89d6ef5a8b9609638666ca95bae51d7'],
+        'runtest': 'pytest',
```

```suggestion
        'runtest': 'pytest --ignore=tests/ignite/contrib --ignore=/tests/ignite/metrics',
        'testinstall': True,
```
Contrib I would think from the name would be perfectly fine to skip for our purposes. Might want to skip tests/ignite/metrics as well or pull in:
 - dill
 - skimage
 - pytorch_fid
 - nltk
 as builddependencies